config/services.yaml

```diff
@@ -1,3 +1,8 @@
+parameters:
+    env(DEFAULT_LOCALE): 'en'
+    kernel.data_dir: '%kernel.project_dir%/data'
+    service_name: 'articles'
```

Comment:
`config/` is copied into the image, so it has this as a default unless the file is overridden by a volume. Are we sure we want this? Should be an environment variable?

Reply:
As it stands using an environment variable won't work. I think resolving https://github.com/libero/content-store/pull/3/files/9fe28b814a5e796befb3e869a372f8a6de9a07ef#diff-935c9b84884d489ccd62bfc1b58a137e will give it a chance. Though this is really a value that shouldn't change...

Reply:
https://github.com/libero/content-store/pull/3/files#diff-9c6968b70659f410eb8de9e4fa30574bR50 was also done to allow mounting in a single file to `prod`.

Reply:
Not sure why it shouldn't change, you could have a `blog-articles` `content-store` instance?

Reply:
I meant once it's available (since it's persisted in the data... unless we set that when serving 🤔).

Reply:
(In https://github.com/libero/content-store/blob/556a45268e33947abad7950ac7ae7798c4850222/.docker/data/article1/1.xml#L5-L9 there should actually be a `<service>articles</service>`.)

Reply:
Will update the ContentApiBundle to allow for prefix-less services.

Reply:
That will be a way to avoid the need for the configuration in the first place.